qpc/cli.py

```diff
@@ -63,18 +63,18 @@
 )
 
 
-class VersionGitAction(argparse.Action):
+class BuildShaAction(argparse.Action):
     """Action to show the current git commit SHA-1."""
 
     def __init__(
         self,
         option_strings,
         dest=argparse.SUPPRESS,
```

Comment:
I thought we would make it visible.

(I'm fine either way)

Reply:
🤷